app/_src/gateway/upgrade-31x.md

```diff
@@ -63,18 +63,18 @@ The following table outlines various upgrade path scenarios to {{page.kong_versi
 
 | **Current version** | **Topology** | **Direct upgrade possible?** | **Upgrade path** |
 | ------------------- | ------------ | ---------------------------- | ---------------- |
-| 2.x–2.7.x | Traditional | No | [Upgrade to 2.8.2.x](/gateway/2.8.x/install-and-run/upgrade-enterprise/) (required for blue/green deployments only), then [upgrade to 3.0.x](/gateway/3.0.x/upgrade/), and then [upgrade to 3.2.x](#migrate-db). |
-| 2.x–2.7.x | Hybrid | No | [Upgrade to 2.8.2.x](/gateway/2.8.x/install-and-run/upgrade-enterprise/), then [upgrade to 3.0.x](/gateway/3.0.x/upgrade/), and then [upgrade to 3.2.x](#migrate-db). |
-| 2.x–2.7.x | DB less | No | [Upgrade to 3.0.x](/gateway/3.0.x/upgrade/), and then [upgrade to 3.2.x](#migrate-db). |
-| 2.8.x | Traditional | Yes | [Upgrade to 3.2.x](#migrate-db). |
+| 2.x–2.7.x | Traditional | No | [Upgrade to 2.8.2.x](/gateway/2.8.x/install-and-run/upgrade-enterprise/) (required for blue/green deployments only), [upgrade to 3.0.x](/gateway/3.0.x/upgrade/), [upgrade to 3.1.x](#migrate-db), and then [upgrade to 3.2.x](#migrate-db). |
```

Comment:
Every single one of these links takes you to the same place. Are they even useful? Maybe we should be linking to version specific topics. That is - upgrading to 2.8 has specific requirements/breaking changes, certainly the upgrade to 3.0 does, etc. 

So maybe we should be linking to:
/gateway/3.0.x/upgrade/, /gateway/3.1.x/upgrade/, and #migrate-db only for the current version (3.2)?

It's tricky for 2.8.x though, since the Enterprise and OSS topics were split then:
https://docs.konghq.com/gateway/2.8.x/install-and-run/upgrade-enterprise/
https://docs.konghq.com/gateway/2.8.x/install-and-run/upgrade-oss/

At that point, maybe we just pick EE and link to that.